src/ExtensionsSample/Program.cs

```diff
@@ -112,5 +116,39 @@ private static void EnsureSampleDirectoriesExist(string rootFilesPath)
 
             File.WriteAllText(Path.Combine(rootFilesPath, "input.txt"), "WebJobs SDK Extensions!");
         }
+
+        private static bool CheckForApiHub(JobHostConfiguration config)
```

Comment:
Recommend removing this - when people enable the sample, they will have configured a valid connection string, so this shouldn't be needed, right?